Assets/MixedRealityToolkit.Examples/Demos/Diagnostics/Scripts/DiagnosticsDemoControls.cs

```diff
@@ -1,50 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think there's also a .meta file that has to be deleted along with this.

Reply:
there should be. let me check why it wasnt

Reply:
it was remvoed. git decided that it was "renamed" to the meta for DiagnosticsSystemVoiceControls